zipline/data/data_portal.py

```diff
@@ -773,19 +773,20 @@ def get_spot_value(self, asset, field, dt, data_frequency):
         if field not in BASE_FIELDS:
             raise KeyError("Invalid column: " + str(field))
 
+        session_label = self.trading_calendar.minute_to_session_label(dt)
```

Comment:
Not sure we can, easily.  We were already normalizing the date below (in minute mode check), and we should be getting the session label anyway.